src/vm/exceptionhandling.h

```diff
@@ -389,15 +389,15 @@ class ExceptionTracker
 
     bool IsStackOverflowException();
 
-#ifdef FEATURE_PAL
+#if defined(FEATURE_PAL) && !defined(CROSS_COMPILE)
```

Comment:
@hseok-oh ah, I can see the problem now. It is because the PAL_SEHException defined in pal.h uses CONTEXT while the m_ptrs.ContextRecord are of T_CONTEXT type. Then the exclusion makes sense since we cannot use T_CONTEXT in the pal.h